src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1961,18 +1971,20 @@ def aks_update(cmd, client, resource_group_name, name,
                load_balancer_outbound_ip_prefixes=None,
                attach_acr=None,
                detach_acr=None,
+               api_server_authorized_ip_ranges=None,
```

Comment:
should we also validate no IPWL if cluster is Basic LB SKU? Or we rely on server side to validate?

I know we still want to support existing Basic LB cluster opted in the IPWL before GA, so not sure if there is a way to distinguish such case from trying to update IPWL for BLB cluster created after GA

Reply:
We validate it on the server side in sync pathway currently.

Don't quite follow the latter part of your comment - my understanding is you must have standard lb for IPW. @ruchikaguptaa is that right?

Reply:
If you have a pre-GA cluster with BLB then you should be able to update the IPWL. The way we do that is check if the AuthorizedIPRange in MC has len > 0 and if BLB then we allow. Any new cluster that doesn't have IPWL enabled but has BLB will be blocked that way. We check that in RP sync, maybe we can skip it here.

Reply:
Sounds good - thanks for the detailed response Ruchika